bluesky/run_engine.py

```diff
@@ -2316,7 +2326,7 @@ async def _stage(self, msg):
         """
         _, obj, args, kwargs, _ = msg
         # If an object has no 'stage' method, assume there is nothing to do.
-        if not hasattr(obj, 'stage'):
+        if not isinstance(obj, Stageable):
```

Comment:
`Stageable` means "has `stage` *and* `unstage`". Is this what we want?

Reply:
See #1507